wheel:
  cmdTimeout: 1.0
  publisherRate: 100.0

  wheels:
    front_right_wheel:
      linkName: front_right_wheel_link
      wheelJointName: front_right_wheel_joint
      cmdTopic: wheel/front_right_cmd
      maxCmd: 1.0
      maxTorque: 10.0

    front_left_wheel:
      linkName: front_left_wheel_link
      wheelJointName: front_left_wheel_joint
      cmdTopic: wheel/front_left_cmd
      maxCmd: 1.0
      maxTorque: 10.0

    rear_right_wheel:
      linkName: rear_right_wheel_link
      wheelJointName: rear_right_wheel_joint
      cmdTopic: wheel/rear_right_cmd
      maxCmd: 1.0
      maxTorque: 10.0

    rear_left_wheel:
      linkName: rear_left_wheel_link
      wheelJointName: rear_left_wheel_joint
      cmdTopic: wheel/rear_left_cmd
      maxCmd: 1.0
      maxTorque: 10.0